macSetup.sh

```diff
@@ -0,0 +1,3 @@
+# Add an alias for the SCE cli in .zshrc file
+echo "./sce.sh completion" >> ~/.bashrc
+source ~/.bashrc
```

Comment:
I believe this is already run by default on most terminals so the user just needs to make the change once in bashrc.